.github/workflows/centreon-collect.yml

```diff
@@ -21,10 +21,10 @@ on:
       - cmake.sh
       - cmake-vcpkg.sh
       - CMakeLists.txt
+      - CMakeListsLinux.txt
```

Comment:
```suggestion
```

Reply:
Why removing CMakeListsLinux here ?

I think I would also add CMakeListsWindows.txt, but maybe it's due to a misunderstanding on my part.

Reply:
Indeed it should be kept, I didn't see this file exist
But is should be added to `push.pathes` too